test/unit/webgl/p5.Camera.js

```diff
@@ -386,13 +386,13 @@ suite('p5.Camera', function() {
 
       // Confirmation that the argument camera and the matrix of the camera
       // that received set() match
-      assert.deepEqual(copyCam.cameraMatrix.mat4, myCam.cameraMatrix.mat4);
+      assert.deepEqual(copyCam.modelMatrix.mat4, myCam.modelMatrix.mat4);
       assert.deepEqual(copyCam.projMatrix.mat4, myCam.projMatrix.mat4);
       // If the set()ed camera is active,
       // the renderer's matrix will also change.
       assert.deepEqual(
-        copyCam.cameraMatrix.mat4,
-        myp5._renderer.uMVMatrix.mat4
+        copyCam.modelMatrix.mat4,
```

Comment:
For these I think we want to compare camera's `cameraMatrix` to the renderer's view matrix